lib/migration-chunks/validation/field.js

```diff
@@ -26,23 +142,14 @@ module.exports = function (chunks, contentTypes = []) {
     const fieldSet = contentTypeFields[contentTypeId] || new Set();
     const fieldRemovals = recentlyRemoved[contentTypeId] || new Set();
 
+    // TODO catch movements across chunks
+    const validationContext = { fieldSet, fieldRemovals, fieldMovements: new Set() };
```

Comment:
Should we really track this across chunks? And if so, are there implications that restrain us from using the same behavior as for the `contentTypeFields` / `recentlyRemoved`?

```js
const fieldMoves = recentlyMoved[contentTypeId] || new Set();
// ...
recentlyMoved[contentTypeId] = fieldMoves;
```

Reply:
I think we should restrict the field movements to only once per field per migration. I can't really think of a case where it would bring any value to move the same field multiple times.

I updated the code to reflect your proposal : )